docs/modeltranslation/registration.rst

```diff
@@ -127,6 +127,40 @@ the translated field ``image`` to the ``NewsWithImage`` model.
     inheritance` broke the project (if you had always subclassed
     ``TranslationOptions`` only, there is no risk).
 
+.. versionadded:: 0.19.0
+
+If you need to specify the languages available for each model, you can do so using the ``languages``
+argument::
+
+    from modeltranslation.translator import translator, TranslationOptions
+    from news.models import News, NewsWithImage
+
+    class NewsTranslationOptions(TranslationOptions):
+        fields = ('title', 'text',)
+
```

Comment:
```suggestion
```

This is absolutely unrelated to this new option, right?

Reply:
Oh, that's example for normal case usage? Do we really need it here?

Reply:
The code, `fields = ('title', 'text',)` doesn't have to be like this. You are right. I agree with your opinion.